### PR TITLE
DEV-3176-new-go-exercise-reversesecondhalf

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

File: tests/reversesecondhalf/main.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import (
+	student "student"
+
+	"github.com/01-edu/go-tests/lib/challenge"
+	"github.com/01-edu/go-tests/solutions"
+)
+
+func main() {
+	table := []string{"hello wold", "", "Kimetsu no Yaiba", "Z", "email123@live.fr", "write  ==> 45m$", "-552", "w58tw7474abc", "fifa world cup `2022`",}
```
Comment (tiagocollot) [inline tests/reversesecondhalf/main.go:11]: Found an extra space in this string "write  ==> 45m$" the extra space is in between "write" and the "==>".

**Review by tiagocollot — CHANGES_REQUESTED** [review]

File: tests/reversesecondhalf/main.go (hunk 1 above)
Comment (zainabdnaya) [reply tests/reversesecondhalf/main.go:11]: This is a test case, So it could be anything since the extra space doesn't make any problem.

**Review by tiagocollot — APPROVED** [review]

**Review by tiagocollot — APPROVED** [review]

**Review by eslopfer — CHANGES_REQUESTED** [review]
Review pending until instructions are clear.

**Review by jotapero — APPROVED** [review]
LGMT :alien:

**Pull request merged**